src/main/php/webservices/rest/Endpoint.class.php

```diff
@@ -89,10 +89,15 @@ public function execute(RestRequest $request) {
     $uri= $this->base->resolve($request->path());
     $conn= $this->connections->__invoke($uri);
 
+    $headers= $request->headers();
+    foreach ($request->cookies()->forURI($uri) as $cookie) {
+      $headers['Set-Cookie'][]= $cookie->name().'='.urlencode($cookie->value());
```

Comment:
https://stackoverflow.com/questions/16305814/are-multiple-cookie-headers-allowed-in-an-http-request

> When the user agent generates an HTTP request, the user agent MUST NOT attach more than one Cookie header field.

Reply:
✅ Done!